### PR TITLE
/README.md: match doc

### PR DESCRIPTION
**Pull request by kfessel**
### Contribution description

take the introductory sentence from our doc

### Testing procedure

read

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,9 @@
 
 The friendly Operating System for IoT!
 
-RIOT is a real-time multi-threading operating system that supports a range of
-devices that are typically found in the Internet of Things (IoT):
+RIOT is an open-source microcontroller operating system, designed to match
+the requirements of Internet of Things (IoT) devices and other embedded devices.
```
Comment (mguetschow) [inline README.md:40]:
```suggestion
the requirements of Internet of Things (IoT) devices and other embedded devices.
```

**Review by mguetschow — APPROVED** [review]
Fine with me :)

**Comment by mguetschow** [comment]
Looks like the trailing whitespace is still present :)

File: README.md (hunk 1 above)
Comment (kfessel) [reply README.md:40]: `git add -u` vs `git commit -a`  `git commit -u` doesn't do the right thing

Comment (kfessel) [reply README.md:40]: it was a relay resilient whit space

**Pull request merged**